flask_project/campaign_manager/test/test_survey.py

```diff
@@ -36,5 +36,10 @@ def test_survey_features(self):
         """
         It should return survey's features.
         """
-        survey = Survey.find_by_name('buildings')
-        self.assertEqual(survey.feature, 'building')
+        surveyPath = os.path.join(
+            os.path.dirname(__file__),
+            '../feature_templates',
+            'buildings.yml'
+        )
+        survey = Survey.find_by_name(surveyPath)
+        self.assertEqual(survey.feature, 'building=yes')
```

Comment:
Again, I'm not sure why the expected value of `survey.feature` here is changing from `building` to `building=yes`, but this seems to me like the correct value to check for, since it's what's in the yaml file.